addons/hr_attendance/static/src/less/hr_attendance.less

```diff
@@ -9,15 +9,15 @@
 
 .oe_hr_attendance_status {
     font-size: 1.2em;
-    margin-right: .3em;
+    margin-right: .1em;
     display: inline-block;
     color: @odoo-brand-secondary;
 
     &.oe_hr_attendance_status_green {
-        color: @brand-success;
+        color: @brand-success !important;
```

Comment:
for community version when we use stat button , "oe_stat_button .o_button_icon" 
overrite "#7c7bad" for all icon. so to change color for attandance state button (for check in and check out state, blue and red.) 
we have to set !important. and this class is defined for hr_attendance only.

note: in enterprise with out !important it's work perfactly..